src/hotspot/share/opto/node.hpp

```diff
@@ -339,9 +339,7 @@ class Node {
  public:
   // Each Node is assigned a unique small/dense number.  This number is used
```

Comment:
While touching the comments anyways:
```suggestion
  // Each Node is assigned a unique small/dense number. This number is used
```